### PR TITLE
52 setup server

### LINKED ISSUE #52: Setup server

### PR DESCRIPTION
**Pull request by KentinPaille**
Initialisation of server api route, creation of the nest back-end and function to get ETH Balance and Token balance on the blockchain BASE

### CODE REVIEW

File: client/.gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+# @generated expo-cli sync-2b81b286409207a5da26e14c78851eb30d8ccbdb
+# The following patterns were generated by expo-cli
+
+expo-env.d.ts
+# @end expo-cli
```
Comment (PaulThblt) [inline client/.gitignore:6]:
```suggestion
# @end expo-cli

```

File: client/package.json [hunk 2]
```diff
@@ -59,6 +59,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.20.0",
+    "@types/node": "^22.9.0",
```
Comment (PaulThblt) [inline client/package.json:62]: Do we need this in the client?

File: server/.prettierrc [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
+  "trailingComma": "all"
+}
```
Comment (PaulThblt) [inline server/.prettierrc:4]:
```suggestion
}

```

File: server/README.md
Comment (PaulThblt) [inline server/README.md:1]: Get rid of this?

File: server/netlify/functions/TokenBalances/TokenBalances.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        return {
+            statusCode: 404,
+            body: JSON.stringify({ error: 'Balance not found' }),
+        };
+    }
+    return {
+        statusCode: 200,
+        body: JSON.stringify({ balance }),
+    };
+};
```
Comment (PaulThblt) [inline server/netlify/functions/TokenBalances/TokenBalances.ts:27]:
```suggestion
};

```

**Comment by PaulThblt** [comment]
Good stuff but I think that we really need to have a separate repo for this

**Review by PaulThblt — CHANGES_REQUESTED** [review]
We need a separate repo

**Comment by bendsp** [comment]
Good work but I second Paul on the separation. We can discuss tomorrow

**Pull request closed without merging**